.github/workflows/ci.yml

```
@@ -53,35 +65,35 @@ jobs:
   # Unit tests - java, go
   go_test:
     uses: 'abcxyz/pkg/.github/workflows/go-test.yml@main' # ratchet:exclude
-    with:
-      go_version: '1.21'
 
   java_test:
     runs-on: 'ubuntu-latest'
     permissions:
       contents: 'read'
       id-token: 'write'
     steps:
-      - uses: 'actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b' # ratchet:actions/checkout@v3
-      - name: 'Set up JDK 11'
-        uses: 'actions/setup-java@1df8dbefe2a8cbc99770194893dd902763bee34b' # ratchet:actions/setup-java@v3
+      - uses: 'actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11' # ratchet:actions/checkout@v4
+
+      - uses: 'actions/setup-java@9704b39bf258b59bc04b50fa2dd55e9ed76b47a8' # ratchet:actions/setup-java@v4
         with:
           java-version: '11'
           distribution: 'adopt'
```

```suggestion
          distribution: 'temurin'
```
Recommended https://github.com/actions/setup-java?tab=readme-ov-file#supported-distributions

They are equivalent, temurin is just the more modern name